tests/src/api_surface_enums.cpp

```
@@ -0,0 +1,53 @@
+// Pins enum values for BlendMode, ImageScaleType, TextAlignment,
+// SpriteSheet::LoopType, and CanyonEventType.  Any rename or reorder will
+// fail to compile.
+
+#include "moth_graphics/moth_graphics.h"
+
+#include <catch2/catch_all.hpp>
+
+using namespace moth_graphics;
+using namespace moth_graphics::graphics;
+
+TEST_CASE("BlendMode enum values are stable", "[api][enums][blend_mode]") {
```

Clang (14.0.6)

[error] variable 'TEST_CASE' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables,-warnings-as-errors)